File: swagger-client/micronaut/src/main/java/io/swagger/model/EmailAddressTypes.java
package io.swagger.model;

import java.util.Objects;
import io.swagger.v3.oas.annotations.media.Schema;
import io.micronaut.core.annotation.Introspected;
import io.micronaut.validation.Validated;
import javax.validation.Valid;
import javax.validation.constraints.*;

/**
 * A flexible single email address that accepts either a plain address string, an RFC 822 named-address string (&#x60;\&quot;Name &lt;email&gt;\&quot;&#x60;), or a structured contact object.
 */
@Schema(description = "A flexible single email address that accepts either a plain address string, an RFC 822 named-address string (`\"Name <email>\"`), or a structured contact object.")
@Validated
@Introspected

public class EmailAddressTypes   {

  @Override
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    if (o == null || getClass() != o.getClass()) {
      return false;
    }
    return true;
  }

  @Override
  public int hashCode() {
    return Objects.hash();
  }

  @Override
  public String toString() {
    StringBuilder sb = new StringBuilder();
    sb.append("class EmailAddressTypes {\n");
    
    sb.append("}");
    return sb.toString();
  }

  /**
   * Convert the given object to string with each line indented by 4 spaces
   * (except the first line).
   */
  private String toIndentedString(java.lang.Object o) {
    if (o == null) {
      return "null";
    }
    return o.toString().replace("\n", "\n    ");
  }
}

File: openapi-client/javascript-closure-angular/API/Client/EmailAddressTypes.js
goog.provide('API.Client.EmailAddressTypes');

/**
 * A flexible single email address that accepts either a plain address string, an RFC 822 named-address string (`\"Name <email>\"`), or a structured contact object.
 * @record
 */
API.Client.EmailAddressTypes = function() {}

/**
 * The email address.
 * @type {!string}
 * @export
 */
API.Client.EmailAddressTypes.prototype.email;

/**
 * Optional display name shown to recipients (e.g. in the `From:` header).
 * @type {!string}
 * @export
 */
API.Client.EmailAddressTypes.prototype.name;

